tests/test_argoverse_stereo_loader.py

```diff
@@ -0,0 +1,68 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+"""Stereo Loader unit tests"""
+import pathlib
+
+import numpy as np
+import pytest
+
+from argoverse.data_loading.argoverse_stereo_loader import ArgoverseStereoLoader
+from argoverse.utils.camera_stats import RECTIFIED_STEREO_CAMERA_LIST
+
+TEST_DATA_LOC = str(pathlib.Path(__file__).parent.parent / "tests" / "test_data" / "stereo")
+
+STEREO_DISPARITY_LIST = ["stereo_front_left_rect_disparity", "stereo_front_left_rect_objects_disparity"]
+
+
+@pytest.fixture  # type: ignore
+def data_loader() -> ArgoverseStereoLoader:
```

Comment:
Let's add docstrings to these functions to describe what they're testing at some point. Not pressing IMO.

Reply:
yeah, agreed. Many of our unit tests have no docstrings.